lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -61,10 +70,16 @@ class ReportUIFeatures {
     this.json = report;
     this._setupMediaQueryListeners();
     this._setupExportButton();
+    this._setupStickyHeaderElements();
     this._setUpCollapseDetailsAfterPrinting();
     this._resetUIState();
     this._document.addEventListener('keydown', this.printShortCutDetect);
     this._document.addEventListener('copy', this.onCopy);
+    this._document.addEventListener('scroll', this._handleStickyHeader);
+    // window.addEventListener is undefined in jest tests.
```

Comment:
which test file(s) fails? This sounds like jsdom wasn't set up correctly

Reply:
yup that was it, fixed.